sdk/monitor/azure-resourcemanager-monitor/src/test/java/com/azure/resourcemanager/monitor/DiagnosticSettingsTests.java

```diff
@@ -73,18 +72,21 @@ public void canCRUDDiagnosticSettings() {
             .withRegion(vm.region())
             .withNewResourceGroup(rgName)
             .withTag("tag1", "value1")
+            .disableSharedKeyAccess()
             .create();
 
-        EventHubNamespace namespace = eventHubManager.namespaces()
-            .define(ehName)
-            // EventHub should be in the same region as resource
-            .withRegion(vm.region())
-            .withNewResourceGroup(rgName)
-            .withNewManageRule("mngRule1")
-            .withNewSendRule("sndRule1")
-            .create();
+        // TODO(v-huizhu2): This request was denied due to internal policy. Local authentication methods are not allowed
+        // Will add back when local authentication is supported.
```

Comment:
Would you help create an issue for this in azure-sdk-for-java repository?
There seems to be other resources that need this feature, as shown in your [summary](https://github.com/Azure/azure-sdk-for-java/issues/47372).